packages/babel-preset/__fixtures__/assets/output.ts

```diff
@@ -0,0 +1,13 @@
+import _asset from './blank.jpg';
+import { __styles } from '@griffel/react';
+import image from './blank.jpg';
```

Comment:
I guess the bundle will just strip the unused asset import ?

Reply:
Yup, correct.